src/webkitgtk/web_context.rs

```diff
@@ -143,6 +143,8 @@ impl WebContextExt for super::WebContext {
   where
     F: Fn(crate::WebViewId, Request<Vec<u8>>, RequestAsyncResponder) + 'static,
   {
+    self.custom_protocols.insert(name.to_owned());
```

Comment:
I think this was a bug unnoticed, and should be instead in `WebContext::register_custom_protocol` for all platforms not just Linux

Reply:
Do other platforms share custom protocol handlers between webviews with the same `WebContext`? If not then why should we disallow this on those platforms?

Reply:
no, other platforms needs to re-register for each webview, but it feels weird being here, it should be in `WebContext::register_custom_protocol` behind an cfg for linux